src/factory.py

```diff
@@ -45,9 +45,7 @@
     "div-zero": x86_model.X86UnicornDivZero,
     "div-overflow": x86_model.X86UnicornDivOverflow,
     "meltdown": x86_model.X86Meltdown,
-    "fault-skip": x86_model.X86FaultSkip,
```

Comment:
I'm using fault skip for debugging some things on AMD, it's quite useful. Let's keep it